### PR TITLE
Add Matthew's OpenNMS bugs

### PR DESCRIPTION
**Pull request by todb-r7**
Poke @mkienow-r7 to eyeball these for errors, then @todb-r7 will send 'em along to upstream.

### CODE REVIEW

File: 2016/6xxx/CVE-2016-6555.json [hunk 1]
```diff
@@ -1,18 +1,99 @@
 {
     "CVE_data_meta": {
-        "ASSIGNER": "cve@mitre.org",
+        "ASSIGNER": "cve@rapid7.com",
+        "DATE_PUBLIC": "2016-09-14T18:18:00.000Z",
```
Comment (mkienow-r7) [inline 2016/6xxx/CVE-2016-6555.json:4]: The blog shows "Nov 15, 2016"

File: 2016/6xxx/CVE-2016-6555.json [hunk 2]
```diff
@@ -1,18 +1,99 @@
...
+    ],
     "data_format": "MITRE",
     "data_type": "CVE",
     "data_version": "4.0",
     "description": {
         "description_data": [
             {
                 "lang": "eng",
-                "value": "** RESERVED ** This candidate has been reserved by an organization or individual that will use it when announcing a new security problem. When the candidate has been publicized, the details for this candidate will be provided."
+                "value": "OpenNMS version 18.0.1 and prior are vulnerable to a stored XSS issue due to insufficient filtering of SNMP trap supplied data. By creating a malicious SNMP trap, an attacker can store an XSS payload which will trigger when a user of the web UI views the data. This issue was fixed in version 18.0.2, released on September 20, 2016."
```
Comment (mkienow-r7) [inline 2016/6xxx/CVE-2016-6555.json]: If you want to be specific replace "views the data" with "views the events list page".

File: 2016/6xxx/CVE-2016-6556.json [hunk 3]
```diff
@@ -1,18 +1,99 @@
...
+    ],
     "data_format": "MITRE",
     "data_type": "CVE",
     "data_version": "4.0",
     "description": {
         "description_data": [
             {
                 "lang": "eng",
-                "value": "** RESERVED ** This candidate has been reserved by an organization or individual that will use it when announcing a new security problem. When the candidate has been publicized, the details for this candidate will be provided."
+                "value": "OpenNMS version 18.0.1 and prior are vulnerable to a stored XSS issue due to insufficient filtering of SNMP agent supplied data. By creating a malicious SNMP 'sysName' response, an attacker can store an XSS payload which will trigger when a user of the web UI views the data. This issue was fixed in version 18.0.2-1, released on September 20, 2016."
```
Comment (mkienow-r7) [inline 2016/6xxx/CVE-2016-6556.json]: 'sysName' or 'sysContact' response

File: 2016/6xxx/CVE-2016-6555.json (hunk 2 above)
Comment (mkienow-r7) [reply 2016/6xxx/CVE-2016-6555.json]: "18.0.2-1" to match the other CVE description

**Review by mkienow-r7 — CHANGES_REQUESTED** [review]

File: 2016/6xxx/CVE-2016-6555.json (hunk 1 above)
Comment (todb-r7) [reply 2016/6xxx/CVE-2016-6555.json:4]: Indeed, but the PR was public earlier so I'm figuring that's the public date of the vuln. (That's the first time anyone knew about it who cared to look.)

Comment (mkienow-r7) [reply 2016/6xxx/CVE-2016-6555.json:4]: Okay, that makes sense!

File: 2016/6xxx/CVE-2016-6555.json (hunk 2 above)
Comment (todb-r7) [reply 2016/6xxx/CVE-2016-6555.json]: Yeah meant to change it here, the final release was just 18.0.2

**Pull request merged**